ui/src/app/pages/access/data-access-requirements.tsx

```diff
@@ -332,55 +330,6 @@ export const renewalRequiredModules: AccessModule[] = [
   duccModule,
 ];
 
-const handleTerraShibbolethCallback = (
-  token: string,
-  spinnerProps: WithSpinnerOverlayProps,
-  reloadProfile: Function
-) => {
-  const handler = withErrorModal({
-    title: 'Error saving NIH Authentication status.',
-    message:
-      'An error occurred trying to save your NIH Authentication status. Please try again.',
-    onDismiss: () => {
-      spinnerProps.hideSpinner();
-    },
-  })(async () => {
-    spinnerProps.showSpinner();
-    await profileApi().updateNihToken({ jwt: token });
-    spinnerProps.hideSpinner();
-    reloadProfile();
-  });
-
-  return handler();
-};
-
-const handleRasCallback = (
```

Comment:
we do not want to remove this

Reply:
mm I see..somehow I thought era is for RAS linking or sth

Reply:
You *can* use RAS for ERA linking, but Shibboleth is more typical.  We also need RAS for the other identiy providers.